src/controller/review.py

```diff
@@ -1,37 +1,66 @@
-from flask import Blueprint, Response
-from ..lib import loadCSV
-from ..db.models import ApplicationsModel
-import os
+from flask import abort, Blueprint, jsonify, request, Response
+from flask_login import current_user, login_required
+
+from src.models.Answer import Answer
+from src.models.Application import Application
+from src.models.lib.Serializer import Serializer
+from src.shared import Shared
 
 review = Blueprint('review', __name__)
 
-_dirname = os.path.dirname(__file__)
+db = Shared.db
+google = Shared.google
+login_manager = Shared.login_manager
 
-@review.route('/configure/upload_csv')
-def doCSV():
-    with open(os.path.join(_dirname, "../lib/sample.csv"), encoding="utf-8") as data:
-        loadCSV.upload_csv(data)
-    return Response(status=200)
 
-@review.route('/review')
-def getApplicant():
-    model = ApplicationsModel.get_applications_model()
-    try:
-        data = model.getLockedApplicantRowForUser(1)
-        if data is None:
-            data = model.getNewLockedApplicantRow(1)
-    except Exception as e:
-        print("exception raised while getting locked applicant")
-        print(e)
-    return Response(str(data))
-
-@review.route('/review/skip')
-def skipApplicant():
-    model = ApplicationsModel.get_applications_model()
+@review.route('/review', methods=['GET'])
+@login_required
+def get_application():
+    """Get application for user"""
+    application = Application.get_application(current_user.get_id())
```

Comment:
is current_user filled by @login_required?

Reply:
current_user is storing a generated user ID, there aren't any ties to the User object in the db.

Reply:
I take that back. In `auth.py`, there is a load_user function that loads in the current user object.

Reply:
flask-login. The answer is flask-login